fb-negotiation/web/main/backend.py

```diff
@@ -0,0 +1,241 @@
+import sqlite3
+import json
+import time
+
+from cocoa.web.main.backend import Backend as BaseBackend
+from cocoa.web.main.backend import DatabaseManager as BaseDatabaseManager
+from cocoa.web.main.utils import Status, Messages
+from cocoa.analysis.utils import reject_transcript
+
+from db_reader import DatabaseReader
+from core.event import Event
+# from analysis.analyze_strategy import StrategyAnalyzer
+
+class DatabaseManager(BaseDatabaseManager):
+    @classmethod
+    def add_survey_table(cls, cursor):
+        cursor.execute(
+            '''CREATE TABLE survey (name text, chat_id text, partner_type text, fluent integer,
+            honest integer, persuasive integer, fair integer, negotiator integer, coherent integer, comments text)''')
+
```

Comment:
only keep negotiator and comments

Reply:
it seems items like partner_type and chat_id are also necessary for bookkeeping?  I imagine partner-type specifies whether that dialogue was human or bot, and chat_id is just a look-up key.  I removed the majority of other columns.